### PR TITLE
use n-makefile, add verify tasks & update docs

### PR DESCRIPTION
**Pull request by bjfletcher**
as well as fixes to make the linters happy

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -1,9 +0,0 @@
-machine:
-  node:
-    version: 4.4.3
```
Comment (andygnewman) [inline circle.yml:3]: Without this Circle is running it's tests on node 0.10.33. Whereas next-article is running on 4.3.1. Maybe advisable to leave this part of the circle.yml in there.

**Comment by andygnewman** [comment]
Nice clean-up. Just a question over the circle.yml.

File: circle.yml (hunk 1 above)
Comment (bjfletcher) [inline circle.yml:3]: @andygnewman thanks, that's good to know. I'll add that back in.

**Comment by andygout** [comment]
(Asking mainly for my understanding) It looks like we don't publish this to `npm` and that's why we don't need the `npm-publish` on deployment [commands](https://github.com/Financial-Times/n-bottle/blob/master/bin/bottle.js#L30-L41) (as given by the deprecated `n-bottle`) in the `circle.yml`?

**Comment by bjfletcher** [comment]
@andygout yes, I have these rules pinned in our dev channel :)
1. If it’s only a bower module OR the circle.yml is set up do a tag build, create a git release/tag
2. otherwise bottle

**Pull request merged**